numba/tests/test_inti_utils.py

```diff
@@ -0,0 +1,42 @@
+import unittest
```

Comment:
This test module has a typo. s/inti/init/

Reply:
indeed